### PR TITLE
[SCL-110] Me endpoint

### PR DESCRIPTION
**Pull request by gridcell**
- includes a list of countries that the user can view

### CODE REVIEW

**Comment by gridcell** [comment]
[country selector](https://trello.com/c/dc2iXPuY/110-country-selector)

File: src/api/resources/me.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from rest_framework.response import Response
+from rest_framework import serializers
+from rest_framework.viewsets import GenericViewSet
+
+from .base import BaseAPISerializer
+from ..models import Profile
+
+
+class ProfileSerializer(BaseAPISerializer):
+    countries = serializers.SerializerMethodField()
```
Comment (saanobhaai) [inline src/api/resources/me.py:12]: Maybe should use module's way of doing DRF integration?:
https://github.com/SmileyChris/django-countries#django-rest-framework
In `StatsSerializer` I did `country = CountryField(country_dict=True)` but of course that's just for an instance

File: src/api/resources/me.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    def get_queryset(self):
+        pass
+
+    def list(self, request, *args, **kwargs):
+        user = self.request.user
+        profile = user.profile
+        if profile is None:
+            raise NotFound()
+
+        return Response(self.serializer_class(profile).data)
```
Comment (saanobhaai) [inline src/api/resources/me.py]: Should this be `self.get_serializer...`?

**Review by saanobhaai — CHANGES_REQUESTED** [review]

File: src/api/resources/me.py (hunk 1 above)
Comment (gridcell) [reply src/api/resources/me.py:12]: I may be mistaken but it looks like I would need to create a Country serializer to do that

Comment (saanobhaai) [reply src/api/resources/me.py:12]: Note for future selves: Using `CountryFieldMixin` worked except it just output a list of code strings. In theory one should be able to combine this with the `CountryField(country_dict=True)` to yield a list of dicts, but as far as I can tell the two don't play nicely together. So, fall back on tried and true.

**Pull request merged**